src/frontends/jax/include/openvino/frontend/jax/node_context.hpp

```diff
@@ -101,6 +101,7 @@ class NodeContext : public frontend::NodeContext {
     }
 
     Output<Node> get_param(const std::string& name) const {
+        FRONT_END_GENERAL_CHECK(m_param_name_to_id.count(name), "No param id corresponding name: ", name, " exist.");
```

Comment:
If I change `context.const_named_param<int64_t>("axes")` (in `src/frontends/jax/src/op/argmax.cpp`) to `context.const_named_param<int64_t>("axis")`, master branch gave the following error message which is not very clear
```
    def convert(self, model: Union[Model, InputModel]) -> Model:
>       converted_model = super().convert(model)
E       openvino._pyopenvino.OpConversionFailure: Check 'is_conversion_successful' failed at src/frontends/jax/src/frontend.cpp:101:
E       FrontEnd API failed with OpConversionFailure:
E       Model wasn't fully converted. Failed operations detailed log:
E       -- argmax with a message:            
E       Exception happened during conversion of operation JaxFrameworkNode_9    
E       unordered_map::at       
E       Summary: 
E       -- Conversion is failed for: argmax 
```
This commit will provide the following better message:
```
    def convert(self, model: Union[Model, InputModel]) -> Model:
>       converted_model = super().convert(model)
E       openvino._pyopenvino.OpConversionFailure: Check 'is_conversion_successful' failed at src/frontends/jax/src/frontend.cpp:101:
E       FrontEnd API failed with OpConversionFailure:
E       Model wasn't fully converted. Failed operations detailed log:
E       -- argmax with a message:
E       Exception happened during conversion of operation JaxFrameworkNode_9
E       Check 'm_param_name_to_id.count(name)' failed at src/frontends/jax/include/openvino/frontend/jax/node_context.hpp:104:
E       FrontEnd API failed with GeneralFailure:
E       No param id corresponding name: axis exist.
E       
E       Summary:
E       -- Conversion is failed for: argmax
```
This will help with future developers' debugging.